updated slider example to fix JSCS issues and added links between examples

jongund opened the pull request.

MichielBijl commented:
Thank you Jon for updating these. Can you please make sure that these files follow our [Contributing to this Repository](https://github.com/w3c/aria-practices#contributing-to-this-repository) rules? More specifically, please [run JSCS and make sure the code conforms to that](https://github.com/w3c/aria-practices#running-jscs-the-javascript-code-style-checker).

jongund commented:
I fixed the JSCS issues and added links between the examples and did another pull request.

Jon


From: Michiel Bijl [mailto:notifications@github.com]
Sent: Tuesday, February 21, 2017 6:50 AM
To: w3c/aria-practices <aria-practices@noreply.github.com>
Cc: Gunderson, Jon R <jongund@illinois.edu>; Author <author@noreply.github.com>
Subject: Re: [w3c/aria-practices] updated slider example to use new aria practices template (#301)


Thank you Jon for updating these. Can you please make sure that these files follow our Contributing to this Repository<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_w3c_aria-2Dpractices-23contributing-2Dto-2Dthis-2Drepository&d=DwMFaQ&c=8hUWFZcy2Z-Za5rBPlktOQ&r=REZD8fc2AwufInstfW3L5jSLVS8bjZtAodDOhat7yAI&m=Gfl-vYeBkNsSUgaUmiSn8zecHEUvmflMviWrYb6gvxw&s=0O3UQsEPtOYwOKnNhsnbrlFLddUH9nsySptsvZyqA50&e=> rules? More specifically, please run JSCS and make sure the code conforms to that<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_w3c_aria-2Dpractices-23running-2Djscs-2Dthe-2Djavascript-2Dcode-2Dstyle-2Dchecker&d=DwMFaQ&c=8hUWFZcy2Z-Za5rBPlktOQ&r=REZD8fc2AwufInstfW3L5jSLVS8bjZtAodDOhat7yAI&m=Gfl-vYeBkNsSUgaUmiSn8zecHEUvmflMviWrYb6gvxw&s=_hOEFc_6bXPQyk9aGU8Yyygg2uxiQD8Z8ltPoVe4eiE&e=>.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_w3c_aria-2Dpractices_pull_301-23issuecomment-2D281335569&d=DwMFaQ&c=8hUWFZcy2Z-Za5rBPlktOQ&r=REZD8fc2AwufInstfW3L5jSLVS8bjZtAodDOhat7yAI&m=Gfl-vYeBkNsSUgaUmiSn8zecHEUvmflMviWrYb6gvxw&s=jBIUCAjex9s5ZORKnle_c7GDVB_IwH5xgDppRlw4fFk&e=>, or mute the thread<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_ABcE7KGu7FdVbOKk3rpo2JQm8oxGZwmLks5ret1mgaJpZM4MGsvQ&d=DwMFaQ&c=8hUWFZcy2Z-Za5rBPlktOQ&r=REZD8fc2AwufInstfW3L5jSLVS8bjZtAodDOhat7yAI&m=Gfl-vYeBkNsSUgaUmiSn8zecHEUvmflMviWrYb6gvxw&s=i7YwKP7WBj30gSOyl330B6HKwT3GPEKhpVrBUFMuiTo&e=>.
